app/views/admin/cms/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :cms_heading do %>
```

Comment:
It's interesting that so keep saying it's a strange way of doing things, because I've based this approach on the rails guides. Thank you for suggesting an alternative, I'll implement this final change to this change to the layout.